### PR TITLE
Skip elixir which is now in Pro

### PR DESCRIPTION
**Pull request by aryx**
test plan:
see related PR in semgrep

### CODE REVIEW

**Comment by aryx** [comment]
see https://github.com/semgrep/semgrep/pull/9308

File: .codemapignore [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# showing just the rules, for pad's codemap
```
Comment (p4p3r) [inline .codemapignore]: Did you actually mean to add this file?

Comment (aryx) [reply .codemapignore]: yes, I've added similar files in other repos to visualize and generate code statistics (like the fact that
we reached recently 1 M LOC, of real code (not generated stuff).

**Review by p4p3r — APPROVED** [review]
LGTM

**Pull request merged**